webpack.config.js

```diff
@@ -62,6 +62,12 @@ export default {
     'eventsource.sharedworker': [
       fileURLToPath(new URL('web_src/js/features/eventsource.sharedworker.js', import.meta.url)),
     ],
+    'katex': [
+      fileURLToPath(new URL('node_modules/katex/dist/katex.min.js', import.meta.url)),
+      fileURLToPath(new URL('node_modules/katex/dist/contrib/auto-render.min.js', import.meta.url)),
```

Comment:
We shouldn't need this auto-render plugin. We can probably take some inspiration [here](https://gitlab.com/gitlab-org/gitlab/-/blob/master/app/assets/javascripts/behaviors/markdown/render_math.js) instead.

Reply:
The auto-render plugin does one thing: It provides ONE function:

```
renderMathInElement
```

If we get rid of that - we have to write all of the code to do that one function.

This seems somewhat ridiculous when it's available already?!

Reply:
I don't know what kind of useless stuff that module does but isn't something like 

```
target.innerHTML = katex.renderToString(text)
```

sufficient to render to HTML? I'd be wary of third-party (contrib) scripts like that. Better to stay first-party only.